devel-common/src/tests_common/test_utils/compat.py

```diff
@@ -64,6 +64,13 @@
     from airflow.utils.xcom import XCOM_RETURN_KEY  # type: ignore[no-redef,attr-defined]
 
 
+try:
+    from airflow.sdk import TriggerRule
+except ImportError:
+    # Compatibility for Airflow < 3.1
+    from airflow.utils.trigger_rule import TriggerRule  # type: ignore[no-redef,attr-defined]
```

Comment:
Since we are doing this, the tests could actually import from here, but that can be a follow-up if you want.

Like this for ex: `from tests_common.test_utils.compat import XCOM_RETURN_KEY`